docs/index.asciidoc

```diff
@@ -518,6 +521,17 @@ overwritten with a warning.
 The data stream type used to construct the data stream at index time.
 Currently, only `logs`, `metrics`, `synthetics` and `traces` are supported.
 
+[id="plugins-{type}s-{plugin}-dlq_custom_codes"]
+===== `dlq_custom_codes`
+
+* Value type is <<number,number>>
+* Default value is `[]`.
+
+List HTTP's error codes that are considered valid to move the events into the dead letter queue.
```

Comment:
While the per-action response codes _mirror_ http status codes, we only handle retries and DLQ routing for failed actions from bulk requests whose HTTP status code is 2xx. This has caused a fair bit of confusion in the past, so I believe we need to be careful to call these out as action response codes and avoid any explicit association with HTTP.

Reply:
I think is sufficient to rephrase it as:
```
List response or single action error codes that...
```
WDYT?

Reply:
This specific line here, but we also have references to HTTP codes throughout.

```suggestion
List single-action error codes from Elasticsearch's Bulk API that are considered valid to move the events into the dead letter queue.
```